fix-small-bugs

Benjamin-GitHub opened the pull request.

Benjamin-GitHub committed:
Update populate_db.sql

gregdyke reviewed (approved):
it would be helpful if your PR or commit message said why you were doing this - some kind of duplicate based on the branch name?

I assume it's related to the bug you found earlier

Benjamin-GitHub committed:
Update populate_db.sql

+there were some names duplicated => fixed
+some same desks, same date, duplicated => solved
+comment months name for easy to read
+change name 'Ali' with 'Ben' ;)

Benjamin-GitHub committed:
Update WeeklyTable.css

+fix small bug in table > hover in small screen size

Benjamin-GitHub committed:
update @media

+set margin-top for the big screen
+update max-width better responsibility

Benjamin-GitHub committed:
Update populate_db.sql

+add empty dates
